blobs.go

```diff
@@ -61,6 +61,12 @@ type Descriptor struct {
 	// depend on the simplicity of this type.
 }
 
+// Descriptor returns the descriptor, to make it satisfy the Describable
```

Comment:
I see. The idea of having an interface there is to allow implementations to type switch on elements being added to the manifest and support specialized behavior. I suppose having `Descriptor` implement `Describable` is a fine exception to make this work. Let's add a comment there, indicating this is the exception and not the rule.